libs/shared/src/packages/table-helpers/src/lib/table/TabTable/TabTable.tsx

```diff
@@ -29,7 +29,10 @@ export const TabTable = <T extends Record<PropertyKey, unknown>>(
   const ref = useRef<HTMLDivElement | null>(null);
   const [_, refHeight] = useResizeObserver(ref);
   const { columns, error, isFetching, packages, resourceName } = props;
-  const gridHeight = packages && packages.length > 30 ? refHeight : "auto-height"
+
+  const refMinHeight = refHeight < 50 ? 500 : refHeight;

```

Comment:
Kunne du ikke finne root causen til problemet?

Reply:
The div that has the ref attached is rendered at the same time as the grid checks the refheight which means it will be 0 because the ref is not attached yet. I could fix this by rendering the div all the time and using ternaries inside the div to render loading/error/content. It might be a better solution? wdyt?

Reply:
Latest commit has the alternative way of fixing it. I guess UI wise its better but code is messier
6957ed86..2fe0714a

Reply:
I think the first one is ok, i can just foresee a situation there 500 is either too much or too little in the future